lockbox-iosTests/AccountStoreSpec.swift

```diff
@@ -0,0 +1,217 @@
+                    if let config = try? FxAConfig.custom(content_base: "https://accounts.firefox.com"),
+                       let fxa = try? FirefoxAccount(config: config, clientId: Constant.fxa.clientID),
+                       let url = try? fxa.beginOAuthFlow(
+                               redirectURI: Constant.fxa.redirectURI,
+                               scopes: [
+                                   "profile",
+                                   "https://identity.mozilla.com/apps/oldsync",
+                                   "https://identity.mozilla.com/apps/lockbox"],
+                               wantsKeys: true) {
+                        expect(urlObserver.events.first!.value.element!.path).to(equal(url.path))
+                    }
+                }
+            }
+
+            describe("profile") {
+                var profileObserver = self.scheduler.createObserver(Profile?.self)
+
+                beforeEach {
+                    profileObserver = self.scheduler.createObserver(Profile?.self)
+                }
+
+                describe("when the keychain has a valid fxa account") {
+                    beforeEach {
+                        self.keychainManager.retrieveResult[KeychainKey.accountJSON.rawValue] = "{\"schema_version\":\"V1\",\"client_id\":\"98adfa37698f255b\",\"config\":{\"content_url\":\"https://accounts.firefox.com\",\"auth_url\":\"https://api.accounts.firefox.com/\",\"oauth_url\":\"https://oauth.accounts.firefox.com/\",\"profile_url\":\"https://profile.accounts.firefox.com/\",\"token_server_endpoint_url\":\"https://token.services.mozilla.com/1.0/sync/1.5\",\"authorization_endpoint\":\"https://accounts.firefox.com/authorization\",\"issuer\":\"https://accounts.firefox.com\",\"jwks_uri\":\"https://oauth.accounts.firefox.com/v1/jwks\",\"token_endpoint\":\"https://oauth.accounts.firefox.com/v1/token\",\"userinfo_endpoint\":\"https://profile.accounts.firefox.com/v1/profile\"},\"login_state\":\"Unknown\",\"oauth_cache\":{\"profile https://identity.mozilla.com/apps/oldsync https://identity.mozilla.com/apps/lockbox\":{\"access_token\":\"42e4bacc8affd23a4ae264fd59ae8ec4a58d5af03b1215daa9cd940f6f80bd9e\",\"keys\":\"{\\\"https://identity.mozilla.com/apps/oldsync\\\":{\\\"kty\\\":\\\"oct\\\",\\\"scope\\\":\\\"https://identity.mozilla.com/apps/oldsync\\\",\\\"k\\\":\\\"VEZDYJ3Jd1Ui0ZVtW8pPHD6LZ48Jd30p-y-PLQQYa0PRcMZtiM6zJO4_I2lxEg__qkxXldPyLiM5PYY9VBD64w\\\",\\\"kid\\\":\\\"1519160140602-WMF1HOhJbtMVueuy3tV4vA\\\"},\\\"https://identity.mozilla.com/apps/lockbox\\\":{\\\"kty\\\":\\\"oct\\\",\\\"scope\\\":\\\"https://identity.mozilla.com/apps/lockbox\\\",\\\"k\\\":\\\"oGGfsZk8xMXtBzGzy2WY3QGPNOTer0VGIC3Uyz9Jy9w\\\",\\\"kid\\\":\\\"1519160141-YqmShzWPQhHp0RNiZs25zg\\\"}}\",\"refresh_token\":\"fd417de6f90683a046070450ead9844a095bee5df934cc0fe3455bd2d41d394a\",\"expires_at\":1531864458,\"scopes\":[\"profile\",\"https://identity.mozilla.com/apps/oldsync\",\"https://identity.mozilla.com/apps/lockbox\"]}}}"
```

Comment:
Using real results is helpful, but we should confirm the account this real results maps to doesn't hold real data for real people.